core/src/main/java/com/datastax/oss/driver/api/core/metadata/schema/KeyspaceMetadata.java

```diff
@@ -245,4 +245,6 @@ default String describeWithChildren(boolean pretty) {
 
     return builder.build();
   }
+
+  boolean shallowEquals(Object other);
```

Comment:
This should be default, in order to avoid a breaking change. You can implement it in terms of the other interface methods.

Reply:
good idea, pushed a fix